### ISSUE TITLE
client to bridge encryption

### ISSUE THREAD

**Issue by weeman1337** [matrix-org/matrix-bifrost#201]
**Is your feature request related to a problem? Please describe.**

I want to connect a XMPP MUC with a matrix room having encryption between:

* XMPP clients and the bridge
* Matrix and the bridge

**Describe the solution you'd like**

For example:

* Matrix encryption on the Matrix side
* OMEMO on the XMPP side with blind trust

This is better than having no encryption at all.  
Out of scope is a real end to end encryption.

If I am running the bridge on a trusted server it does the job.

**Describe alternatives you've considered**

None

**Additional context**

I am not sure whether this is already possible. If yes, I couldn't find anything about that in the docs.

**Comment by Half-Shot** [comment]
No, it's not possible. We've not looked into OMEMO support but it's likely a lot more work that we're willing to commit right now. Would be happy to include support if someone were to write a PR though.

**Comment by keachi** [comment]
There's a OMEMO [plugin](https://github.com/telldus/xmppjs-client-plugins) for xmpp.js.

**Comment by Jieiku** [comment]
This sounds absolutely awesome!
